offchain-modules/packages/x/src/ckb/tx-helper/multisig/deploy.ts

```diff
@@ -151,14 +150,24 @@ function asyncSleep(ms = 0) {
 const main = async () => {
   console.log('\n\n\n---------start init multisig address -----------\n');
   await waitUntilSync();
-  const configPath = './config.json';
-  nconf.env().file({ file: configPath });
-  const config: Config = nconf.get('forceBridge');
-  console.log('config: ', config);
-  await new ForceBridgeCore().init(config);
-  await deploy();
+  const ckbPrivateKey = getFromEnv('CKB_PRIV_KEY');
+  const multisigConfigPath = getFromEnv('MULTISIG_CONFIG_PATH');
+  nconf.file({ file: multisigConfigPath });
+  const multisigScript: MultisigItem = nconf.get('forceBridge:ckb:multisigScript');
+  console.dir(multisigScript, { depth: null });
+  const res = await deploy(ckbPrivateKey, multisigScript);
+  const obj = { forceBridge: { ckb: res } };
+  console.dir(obj, { depth: null });
+  const outputConfigPath = getFromEnv('CONFIG_PATH', '/tmp/force-bridge');
```

Comment:
`path.join` instead of string connection, because of CONFIG_PATH maybe became `/path/to//somewhere` or `C:\path\to\/somewhere`

Reply:
fixed